CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 Changelog
 =========
+1.7.0 (2020-03-01)
```

Comment:
@mparkhe I don't see entries here for the `mlflow gc` command. Were cherry-picked features not included in this autogenerated change log?

cc @smurching

Reply:
Ah, we need to add labels to PRs between the 1.6 & 1.7 releases for them to show up in the changelog (will update the automation to throw when there are unlabeled PRs, & also go back and label the old PRs)

Reply:
Ideally we could also enforce that a PR has a release-notes label before merging

Reply:
yeah! i added labels to most of the PRs -- i must have forgotten the original gc PR